README.md

```diff
@@ -52,18 +52,21 @@ to implement in nearest future.
 - Valid private ssh-key for both clouds which will be using by CloudFerry for data transferring.
 - Credentials of global cloud admin for both clouds.
 - python-dev
-- Fabric ver.>= 1.8.2
-- python-novaclient
-- python-cinderclient
-- python-glanceclient
-- python-keystoneclient
-- python-neutronclient (python-quantumclient)*
+- ecdsa
+- fabric>=1.8.2
+- colorlog
 - ipaddr
+- jinja2
+- mysql-connector-python
+- pycrypto
+- python-cinderclient==1.0.9
+- python-glanceclient==0.12.0
+- python-keystoneclient==0.9.0
+- python-neutronclient==2.3.4
+- python-novaclient==2.15.0
+- python-swiftclient==2.3.1

```

Comment:
I wonder why we even need the list of dependencies here, it simply duplicates `requirements.txt`, meaning we'd have to maintain both `README.md` and `requirements.txt` synchronized (the downsides seem obvious). For this patchset: 

1. Let's only leave relevant modification (the one with `neutron` changed to `quantum/neutron`)
2. The rest of modifications to README.md should go in a separate patch. In that one we should simply remove all the python library requirements and simply replace it with a phrase "all the python requirements are listed in requirements.txt".